my-code/src/components/Button/styled.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import styled from "styled-components"
+
+type ThemeType = {
+  color: {
+    main: {
+      yellow: string,
+      red: string,
+      green: string
+    },
+    grey: {
+      dark: string,
+      grey: string,
+      midgrey: string,
+      lightgrey: string,
+      white: string
+    }
+  },
+  font: {
+    family: string,
+    base: string,
+    family: string,
+    smaller: string,
+    small: string,
+    large: string,
+    display: string
+  },
+  space: Array<string>,
+  transition: string
+}
+
+type PropType = {
+  theme: ThemeType,
+  active: boolean
+}
+
+const RegularButton = styled.button`
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+
+  background: none;
+  background-color: ${(p: PropType) =>
```

Comment:
yeah, I have to admit that flow here feels a bit strange, I think we're ok if we don't flow type the styled files